assets/scripts/Actor/Actor.ts

```diff
@@ -4,10 +4,10 @@ const { ccclass, property } = cc._decorator;
 
 @ccclass
 export default abstract class Actor extends cc.Component {
-  protected isJumping = false;
+  public isJumping = false;
 
   public get isInTheFloor(): boolean {
-    return this.rigidBody.linearVelocity.y === 0 && this.isJumping === false;
+    return !this.isJumping;
```

Comment:
Coloque a parada da linear velocity de volta
```suggestion
    return this.rigidBody.linearVelocity.y !== 0 && !this.isJumping;
```